src/App.vue

```diff
@@ -2,122 +2,170 @@
   <main id="app">
     <section class="inputSection">
       <h1>Linked <span class="title-logo" /> List</h1>
-      <input type="text" @keyup="clearError" v-model="bookmarkTitle" placeholder="Website Title"/>
-      <input type="text" v-model="bookmarkUrl" placeholder="Website URL"v-on:keyup.enter="validateBookmark" />
-      <button @click="validateBookmark" :disabled="isDisabled">Enter</button>
-      <button @click="clearRead" :disabled="!totalRead">Clear Read</button>
+      
+      <input
+        v-model="bookmarkTitle"
```

Comment:
Put directives first `v-` helps understand the intent of the Node immediately when reading the tempalte